### ISSUE TITLE
Unable to authenticate with keycloak oidc

### ISSUE THREAD

**Issue by j-zimnowoda** [redkubes/otomi-core#254]
**Describe the bug:** (a clear and concise description of what the bug is)
I cannot authenticate with keycloak
**To Reproduce**
1. Go to otomi login page and login with oidc
2. Input credentials and follow login instructions 
3. Observe error page "We are sorry... An internal server error has occurred"

**Expected behavior:** (a clear and concise description of what you expected to happen)
I can login

**Screenshots:** (if applicable, add screenshots to help explain your problem)

**Cluster(s):**
 - Cloud: azure
 - Cluster: dta

**Versions:** (seen in console's `/settings`)
 - Core Version: 0.11.53

**Additional context:** (add any other context about the problem here)
Error from keycloak container:
```
[0m[33m07:52:54,252 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-12) SQL Error: 0, SQLState: 23505
[0m[31m07:52:54,252 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-12) ERROR: duplicate key value violates unique constraint "uk_ru8tt6t700s9v50bu18ws5ha6"
  Detail: Key (realm_id, username)=(master,  ) already exists.
```

From the logs keycloak parses username as an empty string and that may be the reason of login error

**Comment by j-zimnowoda** [comment]
This issue may be related with: https://github.com/redkubes/otomi-core/issues/210

**Comment by Morriz** [comment]
Can you confirm this is related to #210 @rawc0der ?

**Comment by Morriz** [comment]
> Can you confirm this is related to #210 @rawc0der ?

@rawc0der ??

**Comment by rawc0der** [comment]
Yes, 
This is related to #210 and I am trying to figure a different solution out in my local cluster. 

The User Registration process is working first time only and does not allow re-registration of existing users. This happens when IDP entry is "replaced" and corrupts internal objects state.

A temp fix is to delete PVC for `keycloak-postgress` pod and sync the `keycloak` and `keycloak-jobs` releases.

**Comment by j-zimnowoda** [comment]
I think have already solved that issue yesterday and implementation 
has been done as well. 
@Morriz @rawc0der can I close this issue ?

**Comment by rawc0der** [comment]
Yes, this issue is fixed, but #210 is still under the radar.
